tests/foreman/ui/test_hw_model.py

```diff
@@ -111,14 +114,20 @@ def test_positive_delete(self):
 
         @expectedresults: Hardware-Model is deleted
         """
+        def not_start_with_digit(value):
+            return not value[0].isdigit()
+
+        valid_names = valid_hw_model_names(
+            not_start_with_digit,
+            'QE Fake Hardware '
+        )
+
         with Session(self.browser) as session:
-            for test_data in valid_hw_model_names():
+            for test_data in valid_names:
+                if 'bugzilla' in test_data:
+                    # while bz 1265150 is not cherry picked html is not valid
+                    # input
+                    continue
```

Comment:
@pondrejk this change is only on the branch, it is not a cherry pick. On master the code is using all valid values, so it will run the right way on the future. Maybe I could add a msg on the bugzilla so when/if @JacobCallahan and BK decide to cherry pick and clone bz, we could change it back. What you think?